internal/services/nginx/nginx_deployment_resource.go

```diff
@@ -76,6 +76,7 @@ func (m DeploymentResource) Arguments() map[string]*pluginsdk.Schema {
 			// docs: <https://docs.nginx.com/nginx-for-azure/billing/overview/>
 			Type:     pluginsdk.TypeString,
 			Required: true,
+			ForceNew: true,
```

Comment:
Could we update the documentation to reflect this?

Reply:
Thanks @katbyte . I have updated the documentation to call this out 👍